discover.go

```diff
@@ -89,9 +89,15 @@ func addSearchPaths(paths []string) {
 			return
 		}
 	}
+
+	rootPath := "/"
+	if val, exists := os.LookupEnv("GHW_CHROOT"); exists {
```

Comment:
Go ahead and change this to PCIDB_CHROOT now that pcidb has been separated from ghw into an independent library.